c2corg_ui/static/js/slideinfo.js

```diff
@@ -4,17 +4,26 @@ goog.require('app');
 
 
 /**
+ * @param {angular.$compile} $compile Angular compile service.
+ * @param {angular.$templateCache} $templateCache service
  * @return {angular.Directive} Directive Definition Object.
+ * @ngInject
  */
-app.slideInfoDirective = function() {
+app.slideInfoDirective = function($compile, $templateCache) {
+
+  var template = function() {
+    return app.utils.getTemplate('/static/partials/slideinfo.html', $templateCache);
```

Comment:
I thought `getTemplate` function was kind of special and we should use it everywhere, because before it didn't work in debug mode.